src/mito2/src/cache.rs

```diff
@@ -179,10 +179,7 @@ impl CacheManager {
     ) -> Option<Arc<SelectorResultValue>> {
         self.selector_result_cache
             .as_ref()
-            .and_then(|selector_result_cache| {
-                let value = selector_result_cache.get(selector_key);
-                update_hit_miss(value, SELECTOR_RESULT_TYPE)
-            })
+            .and_then(|selector_result_cache| selector_result_cache.get(selector_key))
```

Comment:
> non-blocking: This will still increase the cache miss when we disable the selector result cache.

Once selector result cache is disabled, we'd never care about the miss rate of that.